modules/geochemistry/doc/content/modules/geochemistry/database.md

```diff
@@ -0,0 +1,300 @@
+# The chemical database
+
+Author: Andy Wilkins
+
+Notation and definitions are described in [geochemistry_nomenclature.md].
+
+The `geochemistry` module works with chemical databases in the EQ3/6 format.  This page describes the EQ3/6 format, using a database file downloaded from the geochemist workbench site.  See the "Thermo Datasets" chapter of [!cite](gwb_reference) and [https://academy.gwb.com](https://academy.gwb.com)
+
+This page goes through the database file section-by-section.
+
+## Descriptive header
+
+The database file begins with a descriptive header:
+
+```
+dataset of thermodynamic data for gwb programs
+dataset format: jan19
+activity model: debye-huckel
+fugacity model: tsonopoulos
+```
+
+!alert note 
+Only the [Debye-Huckel activity model](activity_coefficients.md) model is currently supported.
+Only the [Spycher-Reed fugacity model](fugacity.md) model is currently supported.
```

Comment:
I have a feeling that the Spycher-Reed model is only suitable/defined for some of the possible gases, and the others will need some other model. I'll try and double check this.

Reply:
Thanks @cpgr